docs/user/library/dicom.rst

```diff
@@ -6,7 +6,7 @@ DICOM Module
 Summary
 *******
 
-.. automodule:: pymedphys.dicom
+.. automodule:: pymedphys_dicom.dicom
```

Comment:
Lots of redundant labelling...

Reply:
That module likely won't be called dicom anymore. I envision there being a swathe of modules under dicom such as:

* pydicom_wrapper
* anonymise
* create
* rtstruct
* rtplan
* collection
* etc...

Just move it like that for now so that it was minimal changes for the split.